### PR TITLE
Fix upload and repost probers tests and lint

### PR DESCRIPTION
**Pull request by sliptype**
### Description

* Fix upload and repost tests
* Add audius eslint config
* Fix lint

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/probers/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  plugins: ['cypress'],
+  extends: ['audius', 'plugin:cypress/recommended']
```
Comment (dylanjeffers) [inline packages/probers/.eslintrc.js:3]: Love that you added this

**Review by dylanjeffers — APPROVED** [review]
So clutch thanks for handling

**Review by Kyle-Shanks — APPROVED** [review]
Nice

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sk-fix-upload-test

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sk-fix-upload-test

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sk-fix-upload-test

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/sk-fix-upload-test

**Pull request merged**